test/object-store/c_api/c_api.cpp

```diff
@@ -4760,6 +4761,80 @@ TEST_CASE("C API - client reset", "[c_api][client-reset]") {
     }
 }
 
+std::unique_ptr<SynchronousTestTransport> test_transport;
```

Comment:
Should all of this be private to the test that uses it or is it intended to be used by other tests?